src/v2/components/ui/hooks/useToast.ts

```diff
@@ -0,0 +1,189 @@
+"use client"
+
+import type { ToastActionElement, ToastProps } from "@/components/ui/Toast"
+import { ReactNode, useEffect, useState } from "react"
+
+const TOAST_LIMIT = 3
+const TOAST_REMOVE_DELAY = 1000000
```

Comment:
Later maybe we should make this accessible from the outside? I'm thinking it would be useful if the storybook version would sustain and could be inspectable.

Reply:
Yup, that's a great idea! Didn't want to spend too much time with it, since we may replace it with Sonner soon! ;)